### DISCUSSION TITLE
Albanian channels

### DISCUSSION THREAD

**Discussion by besgjilani** [iptv-org/community#196] [Q&A]
hi
can you plz try and fix Albanian channels as none of them working 
regards

**Comment by BellezaEmporium** [comment]
Are you sure ? I've tested one random channel and it works. Are you talking about the TVMAK channels, by any chance ?

**Comment by besgjilani** [comment]
> Are you sure ? I've tested one random channel and it works. Are you talking about the TVMAK channels, by any chance ?

ive tested just now and from all of channels 3 are working

**Comment by freearhey** [comment]
@besgjilani Please specify the name of the specific channel that is not working for you.

**Comment by besgjilani** [comment]
RTKI1-2-3-4
klan kosova
tv dukagjini
koha vision
top channel
digitalb channels
tring channels 
alsat m
all that was working now they dont 
regards

**Comment by freearhey** [comment]
Everything still works for me.
![vlcsnap-2021-11-12-14h56m07s536](https://user-images.githubusercontent.com/7253922/141463441-84d042bf-8d27-47b1-899c-094306b2da3e.png)

**Reply by besgjilani** [reply]
![Uploading image.jpg…]()

**Comment by morinashx** [comment]
Everythings working fine, thanks for keeping this updated... you are awesome

**Reply by besgjilani** [reply]
can you be kind and show me wich link are u using plz as to me none are working

**Reply by morinashx** [reply]
All the links are working, your internet speed is low, that's when it becomes an issue to load the channels, you need to have the download speed over 150mps to load the channels Gjilan.

![image](https://user-images.githubusercontent.com/93810402/141711472-14a7aad2-ddc4-448f-9c81-32f1a7cae10a.png)

**Reply by morinashx** [reply]
![image](https://user-images.githubusercontent.com/93810402/141713702-0f34e59f-5227-4fe5-a355-0d161a6c6393.png)

**Comment by besgjilani** [comment]
internet speed is right now 67mbps so i dont thing is slow as other iptv channels are working fine except albanian one

**Reply by BellezaEmporium** [reply]
You might simply have a firewall that is configured to block this host.

**Comment by besgjilani** [comment]
it is quite strange couse other channels are working exept the albanian ones
but thanks anyway

**Comment by besgjilani** [comment]
if you could send me a link the one you using for albanian channels in case im using different one

**Reply by BellezaEmporium** [reply]
I miiiiiiiight have found why.

Try to see if the website doesn't redirect to HTTPS by any chance. HTTPS won't work, but HTTP will.

A link would be http://us.bestvideostreaming.is/8889/playlist.m3u8

![7hd](https://user-images.githubusercontent.com/30985701/141786534-209a8036-4a01-4660-9667-290ecaf3cd91.jpg)

**Comment by morinashx** [comment]
98% of the Albanian channels are not working.

Please someone fix them up 🙏

**Comment by morinashx** [comment]
Anyone working on getting Digitalb please?
Just a few of them working now... 

Please and thank you

**Comment by morinashx** [comment]
Unfortunately, none of the Albanian channels are working.... someone has to intervene and make them functional once again please...

**Comment by morinashx** [comment]
Albo Channels still not working more than 70% are non functional
Very useless

**Comment by BellezaEmporium** [comment]
Spamming for changes won't make the channels changes come faster.

I've issued a new PR that was merged recently, that addresses a few channel changes. Digitalb channels won't be coming back for a while from what I see.
